packages/itwinui-react/src/core/Menu/MenuItem.tsx

```diff
@@ -217,8 +220,20 @@ export const MenuItem = React.forwardRef((props, forwardedRef) => {
     }
   };
 
+  const onClick = (e: React.MouseEvent<HTMLElement>) => {
+    onActivate();
+
+    e.preventDefault();
+    e.stopPropagation();
```

Comment:
It would be useful to have a code comment for every instance of `preventDefault()` and `stopPropagation()`, because both of these are potentially destructive actions.

Reply:
Upon testing, I realized that `e.preventDefault()` and `e.stopPropagation()` were not needed in this case. So, I removed them (https://github.com/iTwin/iTwinUI/pull/1942/commits/5ed937835d330eef33e3294e8d3ce9fe2f416506).